deployment/deploy-app.ps1

```diff
@@ -64,7 +103,7 @@ dataGatewayModule:
 kubectl create secret generic data-gateway-module-secrets-seed --from-literal=dataGatewaySecrets=$dataGatewaySecretsSeed -n $appKubernetesNamespace
 
 # Deploy Flux v2 configuration to install app on kubernetes edge.
-az k8s-configuration flux create -g $AKSClusterResourceGroupName -c $AKSClusterName -t connectedClusters -n edge-framework-ci-config --namespace $appKubernetesNamespace --scope cluster -u https://github.com/suneetnangia/distributed-az-edge-framework --branch main --kustomization name=flux-kustomization prune=true path=/deployment/flux
+az k8s-configuration flux create -g $ResourceGroupName -c $AksCluster2Name -t connectedClusters -n edge-framework-ci-config --namespace $appKubernetesNamespace --scope cluster -u https://github.com/suneetnangia/distributed-az-edge-framework --branch main --kustomization name=flux-kustomization prune=true path=/deployment/flux
```

Comment:
Wonder if we want to add another line here to enable Flux on both AKS1 and AKS2?